src/cozy/wrappers/button-extension/button-extension.jsx

```diff
@@ -0,0 +1,51 @@
+import { useI18n } from 'cozy-ui/transpiled/react/I18n';
+import { default as UIButtonClient } from 'cozy-ui/transpiled/react/PushClientButton';
+import { extensionStoresIcons } from 'cozy/react/components/browserIcons';
+import {
+  extensionStatuses,
+  useExtensionStatus,
+} from 'cozy/react/helpers/extensionStatus';
+import getSupportedPlatforms from 'cozy/react/supportedPlatforms';
+import { detect as detectBrowser } from 'detect-browser';
+import React from 'react';
+import ReactWrapper from '../react-wrapper';
+
+const browser = detectBrowser();
+
+const ButtonClient = ({ ...props }) => {

```

Comment:
We spread here, and then again at l38. Why not `(props)` here and only spread in at line 28?

Reply:
I used it to retrieve a value from props, but I removed this case after a refactoring and forgot to remove the spread syntax.

I fixed this.